### PR TITLE
Run prettier on all files tracked by Git

### PR DESCRIPTION
**Pull request by craigpalermo**
* Updated `.prettierignore` to keep it in sync with `.gitignore`
* Ran `prettier . --write`
* Added `eslint-config-prettier` to Heft eslint config

### CODE REVIEW

File: common/config/rush/command-line.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
       "name": "prettier",
       "commandKind": "global",
       "summary": "Used by the pre-commit Git hook. This command invokes Prettier to reformat staged changes.",
+      "safeForSimultaneousRushProcesses": true,
```
Comment (sdalonzo) [inline common/config/rush/command-line.json:19]: Not sure this is safe

File: packages/core/package.json [hunk 2]
```diff
@@ -115,7 +115,6 @@
     "framer-motion": "^6.5.1",
     "jest-styled-components": "^7.1.1",
     "npm-run-all": "^4.1.5",
-    "prettier": "^2.8.8",
```
Comment (sdalonzo) [inline packages/core/package.json:118]: This is needed afaik for vscode to function correctly

**Review by sdalonzo — CHANGES_REQUESTED** [review]

File: common/config/rush/command-line.json (hunk 1 above)
Comment (craigpalermo) [reply common/config/rush/command-line.json:19]: CleanShot 2023-07-10 at 08 06 00

I got it from whatever's current in the Rush docs

File: packages/core/package.json (hunk 2 above)
Comment (craigpalermo) [reply packages/core/package.json]: Are you setting this to anything in your VSCode settings? I just realized I was getting results inconsistent with our Prettier config when pointing it to `./common/autoinstallers/rush-prettier/node_modules/prettier`. I'll leave this dependency as is just to be safe.

File: common/config/rush/command-line.json (hunk 1 above)
Comment (sdalonzo) [reply common/config/rush/command-line.json:19]: 👍

**Review by sdalonzo — APPROVED** [review]

**Pull request merged**